charts/isaac-app/values.yaml

```diff
@@ -3,6 +3,12 @@
 # Declare variables to be passed into your templates.
 nameOverride: ""
 fullnameOverride: ""
+contentRepo:
+  url: git@github.com:isaaccomputerscience/rutherford-content.git
+  depth: 25
```

Comment:
Defaulting depth to 25 as a safe bet since the difference between "live" and "latest" sha's might be large. I made a note to ask more about this on Thursday.